cmd/awscollector/Dockerfile

```diff
@@ -28,17 +28,36 @@ COPY .git /workspace/.git
 COPY Makefile /workspace/Makefile
 COPY config /workspace/config
 
+ARG TARGETPLATFORM
+RUN echo "Targate platform is $TARGETPLATFORM"
+
 #Copy binary if exist else build binary
-COPY go.* build/linux/aoc_linux_x86_6* /workspace/
-RUN if [[ -f /workspace/aoc_linux_x86_64 ]] ; \
+COPY go.* build/linux/* /workspace/
+RUN if [[ "$TARGETPLATFORM" == "linux/amd64" ]] ; \
     then \
-        echo Copying binary \
-        && mkdir -p /workspace/build/linux \
-        && cp /workspace/aoc_linux_x86_64 /workspace/build/linux/aoc_linux_x86_64 ; \
+        if [[ -f /workspace/aoc_linux_x86_64 ]] ; \
```

Comment:
Should be cleaner when using variables instead of hard code? Such as aoc_build_platform = aoc_linux_x86_64 if the target platform is linux=am64.